appcontainer/SizingModeModel.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {HoistModel, XH} from '@xh/hoist/core';
+import {action, observable, makeObservable} from '@xh/hoist/mobx';
+import {throwIf} from '@xh/hoist/utils/js';
+
+/**
+ * Manage Sizing Mode.
+ *
+ *  @private
+ */
+export class SizingModeModel extends HoistModel {
+
+    @observable sizingMode = null;
+
+    modes = [
+        'large',
+        'standard',
+        'compact',
+        'tiny'
+    ];
```

Comment:
I think it would be worth an enum here - see `FieldType` in `Field.js`.  Devs could still use simple string values, but it formalizes the concept little and type-heavy folks like myself might want to pull it off the enum.

(Also, I have more than once tried to key off of 'medium' instead of 'standard', so I don't think it's entirely self-evident what the values are. 😄)

Reply:
we will do this and then merge